packages/ckeditor5-emoji/src/emojimention.ts

```diff
@@ -61,30 +61,14 @@ export default class EmojiMention extends Plugin {
 		super( editor );
 
 		this.editor.config.define( 'emoji', {
-			dropdownLimit: 6,
-			marker: ':'
+			dropdownLimit: 6
 		} );
 
 		this._emojiDropdownLimit = editor.config.get( 'emoji.dropdownLimit' )!;
-		this._mentionMarker = editor.config.get( 'emoji.marker' )!;
-
 		this._showAllEmojiId = formatEmojiId( SHOW_ALL_EMOJI );
 		this._emojiDatabase = new Database();
 
 		const mentionFeedsConfigs = this.editor.config.get( 'mention.feeds' )! as Array<MentionFeed>;
-		const markerAlreadyUsed = mentionFeedsConfigs.some( config => config.marker === this._mentionMarker );
-
-		if ( markerAlreadyUsed ) {
-			/**
-				 * The `marker` in the `emoji` config is already used by other mention plugin configuration.
-				 *
-				 * @error emoji-config-marker-already-used
-				 * @param {string} marker Used marker.
-				 */
-			logWarning( 'emoji-config-marker-already-used', { marker: this._mentionMarker } );
-
-			return;
-		}
```

Comment:
This is still required, as a user can define its feed that uses `:` as the trigger.

Reply:
Brought the warning back in: [f4670ff](https://github.com/ckeditor/ckeditor5/pull/17611/commits/f4670ff16922110cf983492c2b907b5f4db0490b)